paloma_sdk/client/lcd/api/cw721.py

```diff
@@ -0,0 +1,424 @@
+        """
+        execute_msg = {"approve": {
+            "spender": spender,
+            "token_id": token_id
+        }}
+
+        if expires is not None:
+            execute_msg["approve"]["expires"] = expires
+
+        funds = Coins()
+        tx = await wallet.create_and_sign_tx(CreateTxOptions(
+            msgs=[MsgExecuteContract(
+                wallet.key.acc_address,
+                token,
+                execute_msg,
+                funds
+            )]
+        ))
+        result = await self._c.tx.broadcast(tx)
+        return result
+
+    async def revoke(
+        self,
+        wallet: Wallet,
+        token: str,
+        spender: str,
+        token_id: str
+    ) -> BlockTxBroadcastResult:
+        """Remove previously granted Approval
+        Args:
+            wallet (Wallet): CW721 sender wallet
+            token (str): token address
+            spender (str): token approved address
+            token_id (str): CW721 token id
+        Returns:
+            BlockTxBroadcastResult: Transaction Broadcast Result
+        """
+        execute_msg = {"revoke": {
+            "spender": spender,
+            "token_id": token_id
+        }}
+
+        funds = Coins()
+        tx = await wallet.create_and_sign_tx(CreateTxOptions(
+            msgs=[MsgExecuteContract(
+                wallet.key.acc_address,
+                token,
+                execute_msg,
+                funds
+            )]
+        ))
+        result = await self._c.tx.broadcast(tx)
+        return result
+
+    async def approve_all(
+        self,
+        wallet: Wallet,
+        token: str,
+        operator: str,
+        expires: Optional[dict] = None
+    ) -> BlockTxBroadcastResult:
+        """Allows operator to transfer / send any token from the owner's account.
+            If expiration is set, then this allowance has a time/height limit
+        Args:
+            wallet (Wallet): CW721 sender wallet
+            token (str): token address
+            operator (str): token operator address
+            expires: (dict, optional) {"at_height": height(u64)} / {"at_time":timestamp(nanosecond, u64)}
+        Returns:
+            BlockTxBroadcastResult: Transaction Broadcast Result
+        """
+        execute_msg = {"approve_all": {
+            "operator": operator
+        }}
+
+        if expires is not None:
+            execute_msg["approve_all"]["expires"] = expires
+
+        funds = Coins()
+        tx = await wallet.create_and_sign_tx(CreateTxOptions(
+            msgs=[MsgExecuteContract(
+                wallet.key.acc_address,
+                token,
+                execute_msg,
+                funds
+            )]
+        ))
+        result = await self._c.tx.broadcast(tx)
+        return result
+
+    async def revoke_all(
+        self,
+        wallet: Wallet,
+        token: str,
+        operator: str,
+    ) -> BlockTxBroadcastResult:
+        """Send CW721 token to the other address and run msg

```

Comment:
Is this description correct? This looks like the description for send_nft()

Reply:
fixed.